sdk/react/panels/cast/CastDevicesScreen.js

```diff
@@ -116,10 +127,12 @@ class CastDevicesScreen extends Component {
             marginTop: 20,
           }]}
         >
-          <ListView
+          <FlatList
             style={styles.listViewContainer}
-            dataSource={dataSource}
-            renderRow={(rowData, sectionID, rowID) => this.renderItem(rowData, rowID, castButton, castButtonActive)}
+            data={devices}
+            extraData={this.state}
+            keyExtractor={this.keyExtractor}
```

Comment:
Please define `keyExtractor` function in place, it's not a huge function and doesn't have any specific logic. (I know it's done like that in Facebook examples :)

Reply:
done